lib/awilixManager.ts

```diff
@@ -83,6 +85,21 @@ export function eagerInject(diContainer: AwilixContainer) {
   }
 }
 
+export function getWithTags(diContainer: AwilixContainer, tags: string[]): Record<string, object> {
```

Comment:
`any` would be better than `object` here, as theoretically nothing prevents you from using DI for a boolean or a number